validator/validator.pb.go

```diff
@@ -2527,7 +2527,7 @@ var xxx_messageInfo_HtmlFormat proto.InternalMessageInfo
 // NEXT AVAILABLE TAG: 40
 type TagSpec struct {
 	// The html_format field tells the validator for which html formats
-	// (ie: (<html ⚡> vs <html a4⚡>) this HTML TagSpec is allowed to validate.
+	// (ie: (<html ⚡lang="en"> vs <html a4⚡>) this HTML TagSpec is allowed to validate.
```

Comment:
Please remove `lang="en"` from the comments in this doc.

Reply:
done